repos/fdbt-site/cypress_tests/cypress/support/steps.ts

```diff
@@ -408,35 +420,83 @@ export const editPassengerTypeOtherProductsPage = () => {
     clickElementByText('Back');
 };
 
-export const editStartDateOtherProductsPage = () => {
+export const editPassengerTypeOtherProductsPage = () => {
     clickRandomElementInTable('govuk-table__body', 'product-link');
     getElementById('product-name').should('not.be.empty');
     getElementById('product-status').should('not.be.empty');
     getElementById('fare-type').should('not.be.empty');
+    editPassengerType();
+};
+
+export const editPassengerTypePointToPointPage = () => {
+    cy.get('.active-products').each(($element, index) => {
+        if ($element.text() === '1') {
```

Comment:
What text is this checking is equal to 1?

Reply:
the active products column cell if it has at least 1 services inside it then it should click on that

Reply:
What happens if the text does not equal 1?